shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/main/presentation/MainComponent.kt

```diff
@@ -84,6 +90,14 @@ class DefaultMainComponent(
         navigation.replaceCurrent(configuration)
     }
 
+    private fun submitRating(rating: Rating) {
```

Comment:
While testing, I noticed that if there are several canceled orders in the "view all" section, and I leave a comment, all those orders end up having the same rating and comment

Reply:
Yes you are right, ux mistake :D I will figure out how to deal with that, maybe I can tide it to that order, however, if somebody cancels an order they won't rate the order itself or us :D maybe I will remove the rating button in that case..

Reply:
Modified it :) Now users can rate completed (shipped)-orders which makes much more sense than before :)